### PR TITLE
Remove npm install dependency

### PR DESCRIPTION
**Pull request by torifat**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

POC for https://github.com/Homebrew/brew/pull/1408

### CODE REVIEW

File: Formula/yarn.rb [hunk 1]
```diff
@@ -17,7 +15,7 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    cp_r '.', prefix
```
Comment (MikeMcQuaid) [inline Formula/yarn.rb]: Did you test this locally? This is 100% not going to work because there's e.g. no `libexec`.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
@torifat Please fill out the new PR template, thanks.

**Comment by torifat** [comment]
@MikeMcQuaid Sorry, I thought you just wanted to check it as POC. That's why skipped the template. Just a moment.

File: Formula/yarn.rb (hunk 1 above)
Comment (torifat) [inline Formula/yarn.rb]: Yes, I checked in locally.

- `brew remove yarn`
- `brew install yarn -s`
- `/usr/local/bin/yarn`

screenshot 2016-11-15 20 44 14

Comment (MikeMcQuaid) [inline Formula/yarn.rb]: What's the line after left in for? It appears to be a no-op (and I'm surprised it didn't fail).

Comment (MikeMcQuaid) [inline Formula/yarn.rb]: Yep, that's why you need to `libexec.install buildpath.children`.

Comment (torifat) [inline Formula/yarn.rb]: BTW, it's weird that `bin.install_symlink Dir["#{libexec}/bin/*"]` is working 😕  It shouldn't.

**Comment by MikeMcQuaid** [comment]
Let's leave this open. You can push changes to this PR.

**Comment by torifat** [comment]
Need to figure out a way to handle `node-gyp-bin/`. I will ping once I'm done with it.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by torifat** [comment]
@ilovezfs Sorry, I couldn't manage time to work on it. Please ignore this for the time being.

**Comment by MikeMcQuaid** [comment]
Closing as this PR has stalled.

**Pull request closed without merging**